compiler/optimizer/VPHandlers.cpp

```diff
@@ -5704,6 +5704,13 @@ TR::Node *constrainAcall(TR::ValuePropagation *vp, TR::Node *node)
             TR::VPResolvedClass *newTypeConstraint = NULL;
             if (constraint)
                {
+               // Do nothing if the class of the object doesn't implement Cloneable
+               if (constraint->getClass() && !vp->comp()->fej9()->isCloneable(constraint->getClass()))
+                  {
+                  if (vp->trace())
+                     traceMsg(vp->comp(), "Object Clone: Class constraint on node %p doesn't exist or the class is not cloneable, quit\n", node);
+                  return node;
```

Comment:
Do we really want to include Object.clone on a uncloneable object into the hit/miss debug counters? A miss on such object is expected thus should not be considered as a miss.